src/wp-includes/taxonomy.php

```diff
@@ -2002,7 +2007,7 @@ function wp_get_object_terms( $object_ids, $taxonomies, $args = array() ) {
 	if ( count( $taxonomies ) > 1 ) {
```

Comment:
Yoda skipped: `1 < count( $taxonomies )`

Reply:
Because `$taxonomies` is wrapped in a function call it cannot accidentally be assigned to a value to either way around is valid.

Reply:
See upstream report here:
https://github.com/WordPress/WordPress-Coding-Standards/issues/2028